_posts/en/newsletters/2020-05-20-newsletter.md

```diff
@@ -62,7 +62,25 @@ changes to popular Bitcoin infrastructure software.
 *In this monthly feature, we highlight interesting updates to Bitcoin
 wallets and services.*
 
-FIXME:bitschmidty
+- **Lightning-based messenger application Juggernaut launches:**
+  In a [blog post][juggernaut blog] announcing the first release of Juggernaut,
+  John Cantrell describes how the messaging and wallet features are built using
+  [keysend payments][topic spontaneous payments].
+
+- **Lightning Loop using multipath payments:**
+  Lightning Labs' Lightning Loop product [now uses][lightning loop mpp blog]
+  [multipath payments][topic multipath payments] to convert onchain funds into
```

Comment:
having two links immediately next to each other makes it difficult to see that they're separate links (especially when it's over a link break, as this appears in my browser). Is there a way to rearrange this sentence to avoid that?

Reply:
Agreed! Fixed.